sda-commons-server-spring-data-mongo/build.gradle

```diff
@@ -7,8 +7,8 @@ dependencies {
   api project(':sda-commons-shared-certificates')
 
   // See compatibility matrix:
-  // https://docs.spring.io/spring-data/mongodb/docs/current/reference/html/#compatibility.matrix
-  api 'org.springframework.data:spring-data-mongodb:3.4.18'
+  // https://docs.spring.io/spring-data/mongodb/docs/current-SNAPSHOT/reference/html/#compatibility.matrix
```

Comment:
NIT: That leads to docs about 3-something. I assume, [the latest docs are here](https://docs.spring.io/spring-data/mongodb/reference/).

Reply:
Compatibility information seems to have been removed there.